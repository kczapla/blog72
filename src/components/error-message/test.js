import React from 'react'

import renderer from 'react-test-renderer'
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'

import ErrorMessage from './index'

describe('error message', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<ErrorMessage text={'test error message'}/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('renders with given text', () => {
    render(<ErrorMessage text={'test error message'}/>)
    expect(screen.getByText('test error message')).toBeInTheDocument()
  })
})
