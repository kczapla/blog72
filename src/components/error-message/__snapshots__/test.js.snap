// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error message renders correctly 1`] = `
<section
  className="sc-gsTCUz kSfVUQ"
>
  <span
    className="sc-bdfBwQ gOhIwQ"
  >
    test error message
  </span>
</section>
`;
